examples/examples3d.jl

```diff
@@ -710,3 +710,41 @@ end
 
 
 end
+
+
+@block SebastianM-C ["3d"] begin
+    @cell "DifferentialEquations.jl path animation" [lines, meshscatter, animated, record, diffeq] begin
+        using OrdinaryDiffEq
+
+        function lorenz(du,u,p,t)
+            du[1] = 10.0*(u[2]-u[1])
+            du[2] = u[1]*(28.0-u[3]) - u[2]
+            du[3] = u[1]*u[2] - (8/3)*u[3]
+        end
+
+        u0 = [1.0; 0.0; 0.0]
+        tspan = (0.0, 10.0)
```

Comment:
It simpler to just shorten the timespan. Changing the number of time points in the animation (`ts`) can lead to a poor sampling of the trajectory, due to large jumps.